script/is-it-pretty.js

```diff
@@ -0,0 +1,44 @@
+'use strict'
+
+const prettier = require('prettier');
+const glob = require('glob');
+const fs = require('fs');
+const globPattern = 'app/{src, test}/**/*.{ts, tsx}'
+const prettierOptions = {
```

Comment:
I don't love how many places we have to duplicate our options. Can we just use the defaults?